lib/scss_lint/linter/nestable_selector.rb

```diff
@@ -0,0 +1,74 @@
+module SCSSLint
+  class Linter::NestableSelector < Linter
+    include LinterRegistry
+
+    def check_node(node)
+      visits = Set.new
+
+      node.children.each do |child_node|
+        nestable_node = find_nestable_node child_node, node.children
```

Comment:
We try to following the Jim Weirich philosophy which [includes parentheses when the return value matters](https://twitter.com/jimweirich/status/373814960788221952).

```ruby
nestable_node = find_nestable_node(child_node, node.children)
```